supreme/build.gradle.kts

```diff
@@ -34,6 +33,17 @@ version = supremeVersion
 
 wireAndroidInstrumentedTests()
 
+
+tasks.configureEach {
+    if (name == "linkDebugTestIosX64") {
```

Comment:
So it's not necessary to exclude these tasks in the CI script above? Why are those disabled?

Reply:
see my explanation above. i'll clean up the workflow file and leave a comment in the buildscript